docs/user_manual/retrohunt.md

```diff
@@ -0,0 +1,43 @@
+# Using Retrohunt
+
+## Overview
+
+The Retrohunt feature allows users to scan the historical collection of files in Assemblyline using YARA rules in order to detect early versions of an attack using newer rule sets to understand how an attack has evolved over time.
+
+To start using Retrohunt, make sure you have the correct [configuration](../installation/configuration/retrohunt.md).
+
+If you want to understand how Retrohunt works, a detailed description can be found in the [System Architecture](../overview/architecture.md#yara-back-in-time-retrohunt)
+
+## Retrohunt Interface
+
+Click the "Retrohunt" left navbar link to access the main Retrohunt interface. This page displays all the Retrohunt searches that users have created while taking into consideration their classification. You can filter these searches using your own search parameters, but it does provide quick search actions (on the right side of the search bar) to display jobs that were completed in the last 24h and your own jobs.
+
+![Retrohunt](./images/retrohunt1.png)
+
+## Creating a new Retrohunt search
+
+To create a new search job, click the green "plus" button on the top right of the main interface to open the Create Retrohunt Search page. Enter the following details:
+
+- **Search information**: Provide a description for the search, select which indices to search on (search space) and enter the expiry date (days to live).
+- **Maximum file classification**: This property defines the scope of the included files in the search, but note that even with a higher classification level, users with a lesser classification will not be able to see all the resulting hits after the search has completed.
+- **YARA rule**: When writing a YARA rule, starting to type `rule` will show a snippet that will create a basic rule template. Write a valid rule otherwise submitting the request to create a search will fail.
```

Comment:
```suggestion
- **YARA rule**: When writing a YARA rule, starting to type `rule` will show a snippet that will create a basic rule template. You must write a valid rule, otherwise submitting the request to create a search will fail.
```